packages/tmux/tmux.pacscript

```diff
@@ -0,0 +1,23 @@
+name="tmux"
+version="3.2a"
+url="https://github.com/tmux/tmux/releases/download/${version}/tmux-${version}.tar.gz"
+build_depends="gcc make pkg-config bison"
+depends="libncurses-dev libevent-dev"
+breaks="tmux-git"
```

Comment:
```suggestion
breaks="${name}-git ${name}-bin ${name}-deb ${name}-app"
```

Also add the other package types.

Reply:
Fixed in https://github.com/pacstall/pacstall-programs/pull/361/commits/615b145c09a3c6bd509325f5cba8e977f1e09e50